pkgs/tools/system/swaysettings/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  pkgs,
```

Comment:
```suggestion
{ lib
, stdenv
```

No one puts the whole `pkgs` on the scope. Instead, the packages eventually needed are listed on the input set.

Reply:
Ok corrected